frontend/src/Components/Profile.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react'
+import { Login } from 'Components/Login'
+
+const URL = 'https://auth-login-project.herokuapp.com/users'
+
+export const Profile = (loggedInUser) => {
```

Comment:
this loggedInUser is always undefined, you're not passing any props when mounting this component.
And you don't need it. remove